packages/cli/src/chat/chat-execution-manager.ts

```diff
@@ -64,7 +64,7 @@ export class ChatExecutionManager {
 			name: workflowData.name,
 			nodes: workflowData.nodes,
 			connections: workflowData.connections,
-			active: workflowData.active,
+			active: !!workflowData.activeVersionId,
```

Comment:
Unfortunately we can't simply trust that `activeVersionId` will be included in the `workflowData`. Since the `workflowData` is created when the execution starts, it depends on when the execution has started. If that is before this migration, it will still contain the `active` flag. If it is after, it will only contain the `activeVersionId`. 

The `workflowData` interface should be updated to have either `activeWorkflowId` or `active`. It probably makes sense to add a "data migration" that readds the `active` property every time `workflowData` is read from the DB on the fly, to keep this backwards compatible. And first encapsulate this to a single function somewhere